Upstream repo https://github.com/dannylongeuay/tree-sitter-go-template no
longer exists, which breaks grammar fetching during the build. Use the
source from latest upstream languages.toml instead.

--- a/languages.toml
+++ b/languages.toml
@@ -806,7 +806,7 @@
 
 [[grammar]]
 name = "gotmpl"
-source = { git = "https://github.com/dannylongeuay/tree-sitter-go-template", rev = "395a33e08e69f4155156f0b90138a6c86764c979" }
+source = { git = "https://github.com/ngalaiko/tree-sitter-go-template", rev = "aa71f63de226c5592dfbfc1f29949522d7c95fac" }
 
 [[language]]
 name = "gowork"
